ftplugin/rust.lua

```diff
@@ -1,3 +1,9 @@
+if vim.b.did_ftplugin_rustacean then
+  return
+end
+
+vim.b.did_ftplugin_rustacean = 1
+
```

Comment:
issue: this breaks the plugin. You need to call `vim.lsp.start` for the LSP client to reattach when switching buffers.

```suggestion
```

I've implemented a fix for setting up the `vim.lsp.commands` for rust-analyzer only once in https://github.com/mrcjkb/rustaceanvim/commit/a24df27ef0ab412227ae0d7b40276f0711966834